tests/basic/sharedv4_svc_test.go

```diff
@@ -325,8 +329,9 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 
 		// Skip the test if there are no test-sharedv4 apps
 		testSv4Contexts = getTestSv4Contexts(contexts)
-		if len(testSv4Contexts) == 0 {
-			Skip("No test-sv4-svc apps were found")
+		testSharedV4Contexts = getTestSharedV4Contexts(contexts)
+		if len(testSv4Contexts) == 0 && len(testSharedV4Contexts) == 0 {
+			Skip("No test-sv4-svc or test-sharedv4 apps were found")
 		}
```

Comment:
This is not ideal. If there are only sharedv4 contexts, we will proceed with the sharedv4-svc tests which will be no-ops. But we will mark them as "passed" even thought they were actually "skipped"?

Reply:
as discussed, sharedv4 apps are moved to its own file and context